### PR TITLE
Update free-podcasts-screencasts-ar.md

### PR DESCRIPTION
**Pull request by andronasef**
### Why is this valuable (or not)?

Yes 

### How do we know it's really free?

Check it!

### CODE REVIEW

**Review by Thenlie — APPROVED** [review]
✅ Authors cited, working links, free
✅ Alphabetical order
✅ Not duplicates

File: casts/free-podcasts-screencasts-ar.md [hunk 1]
```diff
@@ -10,3 +10,5 @@
 * [Nakerah Podcast](https://nakerah.net/podcast) - Nakerah Network (podcast)
 * [null++: بالعربي](https://nullplus.plus) - Mohamed Luay, Ahmad Alfy (podcast)
 * [The Egyptian Guy](https://anchor.fm/refaie) - Mohamed Refaie (podcast)
+* [The Weekly Noob](https://theweeklynoob.netlify.app) - Nabil Tharwat (podcast)
+* [Your Big Brother Metwally](https://anchor.fm/metwally) - Ahmed Metwally (podcast)
```
Comment (eshellman) [inline casts/free-podcasts-screencasts-ar.md:14]: Is this your translation? usually we use the title as given by the resource

**Comment by eshellman** [comment]
The book is fine, but could you give the reasons for changing to english language categories instead of the farsi heading that are there?

File: casts/free-podcasts-screencasts-ar.md (hunk 1 above)
Comment (andronasef) [reply casts/free-podcasts-screencasts-ar.md:14]: Ok I will use the original title for this one 
* [Your Big Brother Metwally](https://anchor.fm/metwally) - Ahmed Metwally (podcast)

But for the weekly noob => this original name

**Comment by andronasef** [comment]
@eshellman 

> The book is fine, but could you give the reasons for changing to english language categories instead of the farsi heading that are there?

What do you mean?

File: casts/free-podcasts-screencasts-ar.md (hunk 1 above)
Comment (andronasef) [reply casts/free-podcasts-screencasts-ar.md]: @eshellman Solved Bro!

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**